src/safe-upgrade-tool/run.md

```diff
@@ -0,0 +1,137 @@
+---
+group: software-update-guide
+title: Run the tool
+functional_areas:
+  - Upgrade
+---
+
+The Magento Safe Upgrade Tool (SUT) is a command line (CLI) tool that checks a Magento instance against a specific version by analyzing all the non-Magento modules installed on it.
+
+potential problems that must be fixed in your custom code before attempting to upgrade to a newer version of Magento.
+
+The tool returns a list of errors and warnings that you must address before upgrading to a new version of Magento.
+
+## Use SUT
+
+### Memory limitations
+
+The SUT uses memory on your Magento instance to run—even if you installed the SUT on a different machine. If you are running a system with large third-party modules and files, the SUT requires a high amount of RAM.
```

Comment:
We should give estimate of what "high amount of RAM" is.

Reply:
Do we have info regarding this estimate @sgonzalezmangana or @svera ?

Reply:
@danidelcar it greatly depends on the size of the scanned instance, but a fair assumption is at least 2 GB RAM. Of course, the more the better.

Reply:
thanks @svera

Reply:
Added info